api/app/tests/Behat/bootstrap/v2/Common/IShouldBeOnFrontendTrait.php

```diff
@@ -36,6 +36,14 @@ public function iAmOnNdrReviewPage(): bool
         return $this->iAmOnPage('/ndr\/.*\/review$/');
     }
 
+    /**
+     * @Then I should be on the confirm your details page
+     */
+    public function iAmOnReportConfirmDetailsPage(): bool
+    {
+        return $this->iAmOnPage('/report\/.*\/confirm-details$/');
```

Comment:
You need to be specific here. `.*` is a greedy quantifier you want to use `\d` in there at the very least. I know we have been using `.*` everywhere, but it's a bad pattern and if something goes wrong it makes it harder to debug, specially if we somehow pass a string instead of an int to the url.